deploy/01_role.yaml

```diff
@@ -53,6 +53,13 @@ kind: ClusterRole
 metadata:
   name: configure-alertmanager-operator-view
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes
```

Comment:
I don't see where this (nodes specifically) is used in the rest of the PR here?  Am I missing something?